BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m

```diff
@@ -60,7 +60,7 @@ + (UIImage *)box_iconForFileName:(NSString *)fileName
 
 + (UIImage *)box_genericFileIcon
 {
-    return [UIImage box_iconWithName:@"icon-file-generic"];
+    return [UIImage box_iconWithName:@"other"];
```

Comment:
"other" is a little vague. acknowledging this is for the miscellaneous case, could we name it something a bit more detailed?